hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockOutputStream.java

```diff
@@ -536,9 +530,9 @@ private void setIoException(Exception e) {
           "Unexpected Storage Container Exception: " + e.toString(), e);
       ioException.compareAndSet(null, exception);
     } else {
-      LOG.debug("Previous request had already failed with " + ioe.toString()
+      LOG.debug("Previous request had already failed with {}"
           + " so subsequent request also encounters"
-          + " Storage Container Exception ", e);
+          + " Storage Container Exception ", ioe.toString(), e);
```

Comment:
Avoid eagerly calling `toString()` for log parameters.

```suggestion
          + " Storage Container Exception ", ioe, e);
```